rest-twitter/service/application.py

```diff
@@ -0,0 +1,104 @@
+from flask import Flask, request, jsonify
```

Comment:
write logs

Reply:
For merging purposes I have not done this, next ticket